Dockerfile.debian

```diff
@@ -0,0 +1,31 @@
+FROM python:3.7.3-stretch AS build
+ARG DOCKER_COMPOSE_GITSHA=unknown
+ENV DOCKER_COMPOSE_GITSHA=$DOCKER_COMPOSE_GITSHA
+
+RUN apt-get update && apt-get install -y \
+    curl \
+    gcc \
+    git \
+    libc-dev \
+    libgcc-6-dev \
+    locales \
+    make \
+    openssl && \
+    pip install virtualenv==16.2.0 tox==2.1.1
+# FIXME(chris-crone): virtualenv 16.3.0 breaks build, force 16.2.0 until fixed
+
+WORKDIR /code/
+
+ADD . /code/
+RUN tox --notest
+RUN script/build/linux-entrypoint
+
+ADD docker-compose-entrypoint.sh /usr/local/bin/
```

Comment:
Should these be in the `build` stage? or only in the `runtime` / `deploy` stage?

Reply:
For the build too, since it's used for tests.
Actually the runtime is the one that goes to the hub and the build is the one that will be used by the CI for the tests